src/main/java/com/snowflake/kafka/connector/SnowflakeSinkTask.java

```diff
@@ -135,6 +135,7 @@ public void start(final Map<String, String> parsedConfig)
       .setRecordNumber(bufferCountRecords)
       .setFlushTime(bufferFlushTime)
       .setTopic2TableMap(topic2table)
+      .setMetadataConfig(parsedConfig)
```

Comment:
Do we really need pass all config to Sink? topic2table is extracted from parsedConfig, can we do the same thing on metadata config?

Reply:
Yes, that makes sense. I'll change that